config/jobs/kubernetes-sigs/cluster-api-provider-digitalocean/cluster-api-provider-digitalocean-presubmits.yaml

```diff
@@ -0,0 +1,62 @@
+presubmits:
+  kubernetes-sigs/cluster-api-provider-digitalocean:
+  - name: pull-cluster-api-provider-digitalocean-build
+    always_run: true
+    decorate: true
+    path_alias: sigs.k8s.io/cluster-api-provider-digitalocean
+    spec:
+      containers:
+      - image: golang:1.11
+        command:
+        - make
+        args:
+        - compile
+  - name: pull-cluster-api-provider-digitalocean-test
+    always_run: true
+    decorate: true
+    path_alias: sigs.k8s.io/cluster-api-provider-digitalocean
+    spec:
+      containers:
+      - image: golang:1.11
+        command:
+        - make
+        args:
+        - test-unit
+  - name: pull-cluster-api-provider-digitalocean-verify
+    always_run: true
+    decorate: true
+    path_alias: sigs.k8s.io/cluster-api-provider-digitalocean
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20181024-be2f242dd-master
+        command:
+        - make
+        args:
+        - verify
+  - name: pull-cluster-api-provider-digitalocean-lint
+    always_run: true
+    labels:
+      preset-service-account: "true"
+      preset-bazel-scratch-dir: "true"
+      preset-bazel-remote-cache-enabled: "true"
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20181024-be2f242dd-experimental
+        args:
+        - "--job=$(JOB_NAME)"
+        - "--repo=sigs.k8s.io/$(REPO_NAME)=$(PULL_REFS)"
+        - "--service-account=/etc/service-account/service-account.json"
+        - "--upload=gs://kubernetes-jenkins/pr-logs"
+        - "--scenario=kubernetes_bazel"
+        - "--timeout=15"
+        - "--" # end bootstrap args, scenario args below
+        - "--install=gubernator/test_requirements.txt"
+        - "--test=//..."
+        - "--test-args=--config=lint"
```

Comment:
I'm not entirely sure how to use pod utilities for this. :thinking:

Reply:
you can do something like

```
containers:
      - image: gcr.io/k8s-testimages/kubekins-e2e:v20181024-be2f242dd-experimental
        command:
        - runner.sh
        - kubernetes_bazel.py
        args:
        - "--install=gubernator/test_requirements.txt"
        - "--test=//..."
        - "--test-args=--config=lint"
```

Reply:
@krzyzacy Thanks! I've updated it.